src/test/system_tests/test_utils.js

```diff
@@ -861,6 +862,30 @@ function validate_expiration_header(expiration_header, start_time, expected_rule
     return days_diff === delta_days && rule_id === expected_rule_id;
 }
 
+/**
+ * return the response body and metadata
+ * 
+ * @param {import('@aws-sdk/client-s3').S3} s3_owner
```

Comment:
This name seems weird for a util, owner of what? Let's change to s3_client if it's just a regular client.